app/src/main/java/com/sma2/sma2/ExerciseFragments/ExerciseInstructions.java

```diff
@@ -16,6 +16,8 @@
 import com.sma2.sma2.ExerciseLogic.ScheduledExercise;
 import com.sma2.sma2.R;
 
+import us.feras.mdv.MarkdownView;
+
 public class ExerciseInstructions extends Fragment {
```

Comment:
I think Fragments always need an empty constructor defined. If not they aparently crash sometimes. Not sure if really needed, but just to be safe we should add them

Reply:
Ups that is my class :P I will fix it :D